02.calendar/cal.js

```diff
@@ -0,0 +1,55 @@
+import dayjs from "dayjs";
+import "dayjs/locale/ja.js";
+import minimist from "minimist";
+dayjs.locale("ja");
+
+const args = minimist(process.argv.slice(2));
+let month = args.m ? parseInt(args.m, 10) : null;
+let year = args.y ? parseInt(args.y, 10) : null;
+
+if (month === null) {
+  const today = dayjs();
+  month = today.month() + 1;
+}
+if (year === null) {
+  const today = dayjs();
+  year = today.year();
+}
+
+const first_day = dayjs()
+  .year(year)
+  .month(month - 1)
+  .startOf("month");
+const last_day = dayjs()
+  .year(year)
+  .month(month - 1)
+  .endOf("month");
+
+const currentMonth = first_day.format("M");
+const currentYear = first_day.format("YYYY");
+
+const consoleWidth = 20;
+
+const spacesToAdd = Math.floor(
+  (consoleWidth - `${currentMonth} 月 ${currentYear}`.length) / 2,
```

Comment:
Prettier のチェックが通っていなさそうです。

```suggestion
  (consoleWidth - `${currentMonth} 月 ${currentYear}`.length) / 2
```

Reply:
VSCode内で保存時にPrettier のチェックがされるよう設定し直しました。